src/identity-providers/add/DetailSettings.tsx

```diff
@@ -106,6 +109,19 @@ export const DetailSettings = () => {
   const history = useHistory();
   const { realm } = useRealm();
 
+  const MapperLink = ({ name, mapperId }: IdPWithMapperAttributes) => (
+    <Link
+      to={toIdentityProviderEditMapper({
+        realm,
+        alias: alias!,
+        providerId: providerId,
```

Comment:
If the key and value share the same name the part after the colon can be removed.
```suggestion
        providerId,
```